### PR TITLE
prevent Elig. survey submission with invalid phone

### PR DESCRIPTION
**Pull request by ericcf**
[#108017382]

### CODE REVIEW

File: app/controllers/api/eligibility_surveys_controller.rb [hunk 1]
```diff
@@ -43,7 +42,7 @@ def new_participant
     def add_contact!(participant)
       phone = save_phone(participant)
 
-      return unless phone.valid?
+      fail unless phone.valid?
```
Comment (michaelwehrley) [inline app/controllers/api/eligibility_surveys_controller.rb]: Would a message here be helpful?

Comment (michaelwehrley) [inline app/controllers/api/eligibility_surveys_controller.rb]: i.e., `fail "#{phone.errors.full_messages.to_sentence}" unless phone.valid?`

**Comment by michaelwehrley** [comment]
:+1: :eyeglasses: good!

File: app/controllers/api/eligibility_surveys_controller.rb (hunk 1 above)
Comment (ericcf) [inline app/controllers/api/eligibility_surveys_controller.rb]: Yeah, that would be a good idea here. Thanks!

**Pull request merged**